lib/_debug_agent.js

```diff
@@ -7,7 +7,12 @@ const Buffer = require('buffer').Buffer;
 const Transform = require('stream').Transform;
 
 exports.start = function start() {
-  var agent = new Agent();
+  const agent = new Agent();
+
+  // SmartOS sometimes munges output from `process._rawDebug()`
+  // but Windows can't use `console.error()`.
+  // https://github.com/nodejs/node/issues/2476
+  const log = process.platform === 'win32' ? process._rawDebug : console.error;
```

Comment:
@jbergstroem My public keys are at https://github.com/trott.keys. What else would you need to give me access?